.gitignore

```diff
@@ -31,6 +31,8 @@ CMakeFiles/
 cmake_install.cmake
 CMakeCache.txt
 rtrlib.pc
+cmake-build-debug
```

Comment:
try to not change the Repository's .gitnore, use .git/local/exclude instead

Reply:
I think we should add it, because a lot of people will run into this.

Reply:
what about .idea?

Reply:
also just good to have it imho

Reply:
why not—as long as it doesn’t overgrow. I mean I‘d be happy if we could include .dylib in gitignore and other build artifacts

Reply:
Or we just put into our quality of live pr

Reply:
put it into different branch